app/models/comment.rb

```diff
@@ -1,11 +1,11 @@
 class Comment < ApplicationRecord
-  belongs_to :post, class_name: 'Post', foreign_key: 'postid'
+  belongs_to :posts, class_name: 'Post', foreign_key: 'postid'
```

Comment:
## [OPTIONAL]

- [ ] Kindly correct this syntax as it prevents a comment from being created the correct syntax is `belongs_to :post, class_name: 'Post', foreign_key: 'postid'`. below is the error thrown  by your below syntax
<img width="1418" alt="image" src="https://user-images.githubusercontent.com/65068771/214709496-cf97ace5-c27b-48da-ac63-6d228a6027b7.png">

- [ ] Kindly note that you need to follow the ruby on rails text notation rules  use the `_` to separate your word (eg: `post_id` not `postid`)

Reply:
thank you im going to change it